### DISCUSSION TITLE
Velge hvor mange avganger som skal vises i Holdeplass-visning

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#552] [Ideas]
I visningstypene Kompakt og Kronologisk kan man velge hvor mange av de neste avgangene som skal vises ved å justere størrelsen på boksen de vises i. Men i Holdeplass-visningen, så bare vises de neste ørten avgangene, og det er ingen måte å justere antallet hverken i innstillingene eller ved å endre boksen.

Trenger en måte å gjøre det på :) Takk!

![image](https://user-images.githubusercontent.com/343561/145611980-eac80177-70f8-48de-bc3b-ee88d87e7533.png)

**Comment by draperunner** [comment]
Hei! Kan du forklare use caset ditt og kvifor du ønsker dette? Holdeplass-visninga er meint for å fungere best på éin stoppestad. Kronologisk-visninga er veldig lik, og lar deg justere boksane. Er det avviksteksten du saknar på Kronologisk?

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
